src/lib/components/html-report-page.jsx

```diff
@@ -20,37 +20,55 @@ export type FlowUncoveredLocsProps = {
   uncovered_locs: Array<FlowUncoveredLoc>
 };
 
-/* eslint-disable react/no-unused-prop-types */
-export type FlowCoverageReportProps = {
-  reportType: 'summary' | 'sourcefile',
+export type FlowReportMetaProps = {
+  assets: {
+    css?: Array<string>,
+    js?: Array<string>
+  },
   coverageGeneratedAt: string,
+  htmlTemplateOptions: {
+    autoHeightSource?: boolean,
+  },
+};
+
+export type FlowCoverageSummaryReportProps = {
   assets: {
     css?: Array<string>,
     js?: Array<string>
   },
+  coverageGeneratedAt: string,
+  htmlTemplateOptions: {
+    autoHeightSource?: boolean,
+  },
```

Comment:
I'm not in love with having had to duplicate the fields from `FlowReportMetaProps` here.

I tried to spell it out with the Intersection syntax (https://flow.org/en/docs/types/intersections/) but flow complained.

```javascript
export type FlowCoverageSummaryReportProps = FlowReportMetaProps & {
  coverageSummaryData: FlowCoverageSummaryData,
};
```
would error with:
```bash
src/lib/components/body-coverage-summary.jsx:74
 74:           <FlowCoverageSummaryTable {...props}/>
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ React element `FlowCoverageSummaryTable`
 38: export type FlowCoverageSummaryReportProps = FlowReportMetaProps & B;
                                                                        ^ property `coverageSummaryData`. Property not found in. See: src/lib/components/html-report-page.jsx:38
 74:           <FlowCoverageSummaryTable {...props}/>
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ props of React element `FlowCoverageSummaryTable`

src/lib/components/html-report-page.jsx:70
 70:     <HTMLReportBodySummary {...props}/>
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ React element `HTMLReportBodySummary`
 38: export type FlowCoverageSummaryReportProps = FlowReportMetaProps & B;
                                                                        ^ property `coverageSummaryData`. Property not found in
 70:     <HTMLReportBodySummary {...props}/>
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ props of React element `HTMLReportBodySummary`
```